### ISSUE TITLE
a11y: va-carousel

### ISSUE THREAD

**Issue by kushich** [epicmaxco/vuestic-ui#3264]

**Comment by kushich** [comment]
According to this pattert ([ref](https://www.magentaa11y.com/checklist-web/carousel/)) it should support arrows navigation between slides.
> People who are distracted by movement can pause animations. ([ref](https://www.w3.org/WAI/tutorials/carousels/))

Looks like in our case hover/tap (or even click) should pause animation. Or we should mention in a11y docs section that carousels should be avoided for critical functionality unless there is a strong business case. ([ref](https://shouldiuseacarousel.com/))
